README.md

```diff
@@ -160,6 +160,8 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 
 * **Godot + Appwrite** - A demo app to showcase Godot's integration with Appwrite. [Source Code](https://github.com/lohanidamodar/godot-appwrite)
 
+* **FoodMagic ✨** - An Augmented Reailty Food Delivery app built using Flutter and Appwrite ❤️. [Source Code](https://github.com/Sameerkash/FoodMagic)
```

Comment:
I'd recommend you to add this under the 30DaysOfAppwrite section along with the other projects.